sdk/core/core-compat/src/policies/redirectPolicyOptions.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export interface RedirectPolicyOptions {
+  handleRedirects?: boolean;
+
+  /**
+   * The maximum number of times the redirect URL will be tried before
+   * failing.  Defaults to 20.
+   */
+  maxRetries?: number;
```

Comment:
Is this about the total maximum number of times the request can be retried, or the maximum redirect depth for a single request?